pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -90,9 +95,89 @@ You can test the expose of the metrics by using _curl:_
 curl http://127.0.0.1:2021/metrics
 ```
 
+### Service where clause
+
+Windows service collector will retrieve entire service information in the local box.
+`we.service.where`, `we.service.include`, and `we.service.exclude` can operate to reduce the volume of the service metrics.
+
+To reduce the amount of this metrics, users have to use WHERE clause.
+This syntax is defined in [the WMI Query Language(WQL)](https://learn.microsoft.com/en-us/windows/win32/wmisdk/wql-sql-for-wmi).
+
+Here is how these parameters should work:
+
+#### we.service.where
+
+`we.service.where` is just handled as normal where clause.
+When a user specify the parameter as follows:
+
+```
+we.service.where Status!='OK'
+```
+
+The WMI query should be called as:
+```
+SELECT * FROM Win32_Service WHERE Status!='OK'
+```
+
+Then, WMI mechanism should return the information which has not OK status in this example.
+
+### we.service.include
+
+`we.service.include` is interpreted into where clause.
+If the multiple key-value pairs are specified, the values will be concatenated with `OR`.
+Also, the values contain `%` character, `LIKE` operator should be used in the transrated clause instead of `=` operator.
+When a user specify the parameter as follows:
+
+```python
+we.service.include {"Name":"docker","Name":"%Svc%", "Name":"%Service"}
+```
+
+The parameter should be interpreted as:
+```
+(Name='docker' OR Name LIKE '%Svc%' OR Name LIKE '%Service')
+```
+
+The WMI query should be called with the translated parameter as:
+
+```
+SELECT * FROM Win32_Service WHERE (Name='docker' OR Name LIKE '%Svc%' OR Name LIKE '%Service')
+```
+
+### we.service.exclude
+
+`we.service.where` is interpreted into where clause.
+If the multiple key-value pairs are specified, the values will be concatenated with `AND`.
+Also, the values contain `%` character, `LIKE` operator should be used in the translated clause instead of `!=` operator.
```

Comment:
```suggestion
Also, if the values contain `%` character then a `LIKE` operator will be used in the translated clause instead of the `!=` operator.
```